apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingRoutes.kt

```diff
@@ -339,7 +344,7 @@ data class LagretHendelser(
 
 data class ManueltOpphoerResponse(val behandlingId: String)
 
-data class VirkningstidspunktRequest(@JsonProperty("dato") private val _dato: String) {
+data class VirkningstidspunktRequest(@JsonProperty("dato") private val _dato: String, val begrunnelse: String?) {
```

Comment:
Noen grunn til at vi lar den være nullable? Hvis den uansett valideres for å ikke være null

Reply:
Kanskje for bedre respons?

Reply:
egentligen mest for å gi tilbake 404 hvis begrunnelse ikke sendes med. ikke noe spesielt mer!

Reply:
404 er vel også litt feil - burde være en 400 hvis det mangler ?

Reply:
hehe mente 400, d.v.s BadRequest

Reply:
men endrer til ikke nullable, den bør alltid komma med.